cmd/preview/list.go

```diff
@@ -72,16 +86,73 @@ func executeListPreviews(ctx context.Context, opts ListFlags) error {
 		}
 		return fmt.Errorf("failed to get preview environments: %s", err)
 	}
-	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
-	fmt.Fprintf(w, "Name\tScope\tSleeping\tLabels\n")
+	switch opts.output {
+	case "json":
+		previewListOutput, err := getPreviewOutput(ctx, opts)
+		if err != nil {
+			return err
+		}
+		bytes, err := json.MarshalIndent(previewListOutput, "", " ")
+		if err != nil {
+			return err
+		}
+		oktetoLog.Println(string(bytes))
+	case "yaml":
+		previewListOutput, err := getPreviewOutput(ctx, opts)
+		bytes, err := yaml.Marshal(previewListOutput)
+		if err != nil {
```

Comment:
we need to check if `getPreviewOutput` failed before marshalling `previewListOutput` - if the previous function failed, this most likely will result in some panic

Reply:
Resovled